.ebextensions/01_loadbalancer-terminatehttps.config

```diff
@@ -1,6 +0,0 @@
-option_settings:
```

Comment:
Removing this file is wrong. This is needed for when we deploy to production. The [deployment](https://github.com/NYPL-discovery/discovery-front-end/blob/master/DEPLOYMENT.md) doc has information about how to deploy. You're suppose to merge this working branch into `dev-eb-deploy` and deploy that branch instead of deploying this branch.